src/storefront/src/templates/data/_data.pug

```diff
@@ -84,7 +84,11 @@
   }, {
     name: 'Contact',
     href: 'contacts.html'
-  }]
+  },
+  {
+    name: 'Reviews',
+    href: 'reviews.html'
+   }]
```

Comment:
Since this is to be part of a canary deployment exercise, this "tab" needs to support enablement when a `--build-arg` is used (you can define this).

Reply:
I see the option here would be to remove the details from _data.pug and write something like this in beta.js

<div uk-alert class="uk-alert-warning">
        <p>You are viewing a beta version of MuShop: <span class="uk-text-bold">v{{ver}}</span>
        <br>
        Features enabled on this version are :</a>
        <ul>
          <li>
            <a href="reviews.html" mu-route>Review</a>
          </li>
        </ul>
        </p>  
      </div>`

This would be the much simpler solution.